prow/janitor.sh

```diff
@@ -112,3 +108,33 @@ for service in $ENDPOINTS_SERVICES ; do
 
 done
 echo "Done cleaning up Endpoints Services"
+
+# Load balancer target pools
+# Source: https://gist.github.com/prasvats/b2a4e33ad12b40191dd9d7e222d1abde/
```

Comment:
Just want to make sure the license etc is fine?

Reply:
Unfortunately it is not licensed, maybe we cannot use it then.